fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraRelaxedLdpIT.java

```diff
@@ -0,0 +1,418 @@
+            assertEquals(OK.getStatusCode(), getStatus(response));
+            containedBinaryDescriptionContentType = response.getFirstHeader("Content-Type");
+            containedBinaryDescriptionBody = EntityUtils.toString(response.getEntity());
+        }
+
+
+        // delete the container and its tombstone
+        try (CloseableHttpResponse response = execute(new HttpDelete(containerURI))) {
+            assertEquals(NO_CONTENT.getStatusCode(), getStatus(response));
+            try (final CloseableHttpResponse r2 = execute(new HttpGet(containerURI))) {
+                final Link tombstone = Link.valueOf(r2.getFirstHeader(LINK).getValue());
+                assertEquals("hasTombstone", tombstone.getRel());
+                assertEquals(NO_CONTENT.getStatusCode(), getStatus(new HttpDelete(tombstone.getUri())));
+            }
+        }
+
+
+        // post the container from the export
+        final String containerRdf = filterRdf(containerBody, CREATED_BY, CREATED_DATE, LAST_MODIFIED_BY,
+                LAST_MODIFIED_DATE, createProperty("http://purl.org/dc/elements/1.1/title"));
+        try (CloseableHttpResponse response = postResourceWithTTL(containerURI, containerRdf)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+        }
+
+        // post the binary then patch its metadata to match the export
+        try (CloseableHttpResponse response = postBinaryResource(containerURI,
+                containedBinaryURI.substring(containerURI.length() + 1), "content")) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+        }
+        final String sparqlUpdate;
+        try (CloseableDataset d = getDataset(new HttpGet(containedBinaryDescriptionURI))) {
+            final Model model = createDefaultModel();
+            model.read(new ByteArrayInputStream(containedBinaryDescriptionBody.getBytes()), "", "N3");
+            final GraphDifferencer diff = new GraphDifferencer(model,
+                    iteratorToStream(d.getDefaultModel().listStatements()).map(statement -> statement.asTriple()));
+            sparqlUpdate = buildSparqlUpdate(diff.difference(), diff.notCommon(), CREATED_BY, CREATED_DATE,
+                    LAST_MODIFIED_BY, LAST_MODIFIED_DATE);
+        }
+        final HttpPatch patch = new HttpPatch(containedBinaryDescriptionURI);
+        patch.setHeader("Content-type", "application/sparql-update");
+        patch.setEntity(new StringEntity(sparqlUpdate));
+        try (CloseableHttpResponse response = execute(patch)) {
+            assertEquals(NO_CONTENT.getStatusCode(), getStatus(response));
+        }
+
+        // Because the creation of the contained resource resulted in an implicit modification to
+        // the triples on the parent, we need to overwrite them again...
+        assertEquals(NO_CONTENT.getStatusCode(), getStatus(putResourceWithTTL(containerURI, containerRdf)));
```

Comment:
Currently the FedoraLdp (http-client-api) class makes two calls to create a new resource when triples are provided.  That makes it very difficult to avoid this call.  That said, it might double the time required for imports if we don't change it.  Let me know whether it's worth reworking that.